server/src-lib/Hasura/RQL/DDL/Metadata/Types.hs

```diff
@@ -425,7 +425,24 @@ replaceMetadataToOrdJSON ( ReplaceMetadata
                   ] <> catMaybes [maybeCommentToMaybeOrdPair comment]
 
     scheduledTriggerQToOrdJSON :: CreateScheduledTrigger -> AO.Value
-    scheduledTriggerQToOrdJSON = AO.toOrdered
+    scheduledTriggerQToOrdJSON (CreateScheduledTrigger name webhook schedule payload retryConf headers) =
+      AO.object $ [ ("name", AO.toOrdered name)
+                  , ("webhook", AO.toOrdered webhook)
+                  , ("schedule", AO.toOrdered schedule)
+                  ] <> catMaybes [ maybeAnyToMaybeOrdPair "payload" AO.toOrdered payload
+                                 , maybeAnyToMaybeOrdPair "retry_conf" AO.toOrdered (maybeRetryConfiguration retryConf)
+                                 , maybeAnyToMaybeOrdPair "headers" AO.toOrdered (maybeHeader headers)]
+      where
+        maybeRetryConfiguration :: RetryConfST -> Maybe RetryConfST
+        maybeRetryConfiguration rc = if retryConf == defaultRetryConfST
```

Comment:
either remove the first parameter so that you don't have to declare `rc` or use `rc` within the body of the function for comparison.